modules/tinymce/CHANGELOG.md

```diff
@@ -16,6 +16,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Line separator scrolling in floating toolbars. #TINY-8948
 - A double bottom border appeared on inline mode editor for the `tinymce-5` skin. #TINY-9108
 - The editor header showed up even with no menubar and toolbar configured. #TINY-8819
+- Dragging over CEF element now works correctly #TINY-8881
```

Comment:
Not sure if it's all that better but maybe since CEF is kind of internal name:

```suggestion
- Dragging a noneditable element before or after another noneditable element now works correctly. #TINY-8881
```

Reply:
done